frontend/src/components/pongGame/ActiveGameRow.vue

```diff
@@ -0,0 +1,30 @@
+<template>
```

Comment:
the idee of the element/component OverViewWithMidline is that you can use component from a 'page component' and 'feed it a data array with left and right.

<OverviewWithMidline :data-array="dataArray" />

this.dataArray = [
				{ left: 'intra name', right: this.user.name },
				{ left: 'member since', right: this.user.created_at },
				{ left: 'wins', right: 1 },
				{ left: 'losses', right: 190 },

you can look at profileview for an example